frontend/src/pages/home-page.tsx

```diff
@@ -40,18 +40,18 @@ function HomePage() {
             </button>
           </div>
           <div className="flex max-w-5xl flex-1 flex-col justify-end pb-8">
-            <h1 className="text-4xl font-bold">Journey Beyond Horizons</h1>
-            <p className="my-4 text-xl">
+            <h1 className="text-4xl font-bold dark:text-white">Journey Beyond Horizons</h1>
+            <p className="my-4 text-xl dark:text-white">
               Dive into the World of Travel with Stories That Transport You to Far-Off Lands.
```

Comment:
Can we avoid using it multiple times by giving `dark:text-white` to the top parent div ?

Reply:
@SankalpHaritash21, please dont mark the conversation as resolved without actually resolving it. It's okay to ask us if you don't know or find it challenging we are here to help, by marking it falsely resolved you are just disrespecting the reviewer's time of reviewing the code. This is the second time I'm able to see this. If this follows I might have to decline the PR all together and assign it to some other potential candidate.

The analogy can be applied as it is just like I am rejecting your PR without any acknowledgement

Reply:
@krishnaacharyaa i thought I resolved it I will update it. And this will not happen again

Reply:
@krishnaacharyaa  I don't have any intention to disrespect the reviewer.